src/app/cli/src/error.rs

```diff
@@ -249,7 +249,7 @@ pub struct NotInMultiTenantWorkspace;
 #[derive(Error, Clone, PartialEq, Eq, Debug)]
 #[error("Multi-tenant reference is unexpected in single-tenant workspace: {dataset_ref}")]
 pub struct MultiTenantRefUnexpectedError {
-    pub dataset_ref: opendatafabric::DatasetRef,
+    pub dataset_ref: String,
```

Comment:
Why not use `DatasetRefPattern` here? Since `DatasetRef` type is a strict subset of `DatasetRefPattern` - it can be used to represent both.